packages/manager/src/features/VPC/VPCEmptyStateData.tsx

```diff
@@ -0,0 +1,15 @@
+import type {
+  ResourcesHeaders,
+  ResourcesLinks,
+} from 'src/components/EmptyLandingPageResources/ResourcesLinksTypes';
+
+export const headers: ResourcesHeaders = {
+  description: '',
+  subtitle: 'Create a Private and isolated network.',
```

Comment:
Just noting on my quest to squash all unnecessarily capitalized words:
Is there a reason we'd want to capitalize the P  here? I think this should be "private" here and [L88](https://github.com/linode/manager/pull/9467/files#diff-22bad342acc3f59f48a6b1b84a0fec7e4c49a90bb08eb70db74c188d745a0709R88) of the landing page test.
```suggestion
  subtitle: 'Create a private and isolated network.',
```

Reply:
Nope no reason, good catch! 🔍